web/src/components/StateDash.js

```diff
@@ -154,10 +154,7 @@ const StateDash = () => (
               { day: 30, title: 'Performance Progress CoP' }
             ]}
           />
-          <Events
-            month="July"
-            events={[{ day: 7, title: 'Annual APD Due' }]}
-          />
+          <Events month="July" events={[{ day: 7, title: 'Annual APD Due' }]} />
```

Comment:
prettier being prettier (PBP)

Reply:
Prettier don't think it be like it is but it do.